include/picongpu/particles/filter/filter.hpp

```diff
@@ -19,5 +19,6 @@
 
 #pragma once
 
+
```

Comment:
pls do not add a double newline before & after `#pragma once` but a double newline after the last include